release-config.ts

```diff
@@ -0,0 +1,4 @@
+export default {
+  commentOnReleasedPullRequests: false,
+  skipLabels: ['skip-changelog'],
```

Comment:
```suggestion
```

`skip-changelog` is already part of the default config or do you want to remove the other labels?

Reply:
Ah ok I was not aware of it